docs/learn-faq.md

```diff
@@ -81,6 +81,88 @@ This script helps you quickly identify which validator is the lowest-staked and
 
 It is also possible to copy both `own stake` and `other stake` on the Staking page to a spreadsheet and then do a simple calculation.
 
+## Myths
+
+Below we present some myths about Polkadot that we've seen in the community.
+We're not absolutely sure what the origin of these myths are -- maybe they arise
+from people who have not taken the time to understand Polkadot. Regardless,
+we have collected a rebuttal for each of the common myths below. Feel free to
+link out to this page if you see one of these myths propagated on discussion platforms
+of the internet. 
+
+### Polkadot is an "Ethereum-killer"
+
+Some people position blockchains antagonistically against each other, claiming
+that all of the new blockchains are by default "Ethereum killers". The line of
+thinking is that there is a limited market share for blockchain platforms and
+therefore all projects are competing and vying for the lion's share. There may
+be some truth to this in the wide ecosystem of blockchain projects, but not all
+of them.
+
+Polkadot and Ethereum have significant differences in their technology, governance
+ideology, and community that instead forms a complimentary whole instead of a
+divided pie. Polkadot is a network for parachain applications, which may likely
+be domain-specific blockchains. Ethereum is instead a smart contract chain that
+focuses on maximizing the decentralization of its architecture. For more on the
+differences between the two chains see the more comprehensive [comparison article](learn-comparison-ethereum).
+
+In order to be an Ethereum killer, Polkadot would have to be actively working
+against the progression of the Ethereum technology and ecosystem, which is not
+true. Instead, Web3 Foundation, the holders of the treasury from the DOT sales, 
+promotes and funds technology that benefits the web3 world as a whole.
+
```

Comment:
I would end this section with examples of how exactly we're not working against Eth, but with it.

- emphasize some collaborations
- indicate that bridging the technologies is possible and that a merger - not competition - of the communities is likely
- emphasize research efforts that benefit both platforms, and if possible note contributions from W3F's research to the Eth2 progress so far.
- emphasize that Polkadot does not come with smart contracts and as such does not compete with Ethereum. Rather, it makes it possible for Ethereum alternatives to exist in a secure way, possibly bridging many different smart contract platforms that are currently growing in isolation (Tezos being notable)